src/utils/debug.cpp

```diff
@@ -144,8 +148,59 @@ void addAttachment(Attachment::AttachmentType type)
 
 }   // addAttachment
 
+// ----------------------------------------------------------------------------
+/** Computes the distance between two points represented by one vector3df each */
+float distance(core::vector3df one, core::vector3df two)
+{
+    return sqrt(pow(one.X - two.X, 2) + pow(one.Y - two.Y, 2) + pow(one.Z- two.Z, 2));
+}
+
```

Comment:
core::vector3df already contains a method named "getDistanceFrom" that does this, so this could could be slightly simplified by using this method